tests/core/RequestHandler.spec.ts

```diff
@@ -234,8 +230,9 @@ describe('RequestHandler', () => {
     const [, anySigningPrivateKey] = await Jwk.generateEs256kKeyPair();
     const [, anyNextUpdateCommitmentHash] = OperationGenerator.generateCommitRevealPair();
```

Comment:
Sanity: I think `OperationGenerator.generateCommitRevealPair()` should be removed completely with this change?